tools/gethostlatency.bt

```diff
@@ -19,6 +19,8 @@
  * 08-Sep-2018	Brendan Gregg	Created this.
  */
 
+config = { warn_missing_probes = 0; }
```

Comment:
I'm not saying we should change this but I guess we should be aware that this will break this script on older bpftrace versions that don't support the config syntax. I guess that's fine as people can just use an older version of the repo. -- thinking out loud.

Reply:
Yeah, since we release the scripts in lockstep with the tool, I think it's fine if the tools do not work for previous versions.